doc/users/next_whats_new/imshow_extent_units.rst

```diff
@@ -0,0 +1,26 @@
+The *extent* parameter of imshow and set_extent functions can now be expressed with units
+-----------------------------------------------------------------------------------------
```

Comment:
`set_extent` is implied here, no need for it in the title.
```suggestion
The *extent* of ``imshow`` can now be expressed with units
----------------------------------------------------------
```

Reply:
done